jupyterlab_gitsync/jupyterlab_gitsync/handlers.py

```diff
@@ -0,0 +1,71 @@
+import base64
+import json
+import re
+import tornado.gen as gen
+import os
+import subprocess
+
+from collections import namedtuple
+from notebook.base.handlers import APIHandler, app_log
+
+from jupyterlab_gitsync.version import VERSION 
+
+
+
+class SyncHandler(APIHandler):
+
+  """
+  Implements all synchronization operations
+  * uses git-sync-changes bash script
+  """
+
+  @gen.coroutine
+  def post(self, *args, **kwargs):
+    recv = self.get_json_body()
+    path = recv['path'] if recv['path'] else '.'
+    ex_path = recv['ex_path'] if recv['ex_path'] else None
```

Comment:
A good fallback value here would be `['git', 'sync-changes']`.

I.E. if, for some reason, we don't pass in the bundled version of the script, then fallback to relying on the user to have it installed in their `PATH`.

Reply:
Will be resolved in future PR.